<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSafeUExplorer</class>
 <widget class="QMainWindow" name="CSafeUExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSafeUExplorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="tvGlobal"/>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="UverticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SafeUDisk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUDisk">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="tvSafeUDisk">
          <property name="lineWidth">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget2">
       <layout class="QVBoxLayout" name="LverticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>LocalDisks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLocal">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="twLocal"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDesktop">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/desktop.png</normaloff>:/images/desktop.png</iconset>
   </property>
   <property name="text">
    <string>Desktop</string>
   </property>
   <property name="toolTip">
    <string>Desktop</string>
   </property>
  </action>
  <action name="actionUpFolder">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/upfolder.png</normaloff>:/images/upfolder.png</iconset>
   </property>
   <property name="text">
    <string>UpFolder</string>
   </property>
   <property name="toolTip">
    <string>UpFolder</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionChgPassword">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/chgpasswd.png</normaloff>:/images/chgpasswd.png</iconset>
   </property>
   <property name="text">
    <string>ChgPassword</string>
   </property>
   <property name="toolTip">
    <string>ChgPassword</string>
   </property>
  </action>
  <action name="actionFormatUDisk">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/format.png</normaloff>:/images/format.png</iconset>
   </property>
   <property name="text">
    <string>FormatUDisk</string>
   </property>
   <property name="toolTip">
    <string>FormatUDisk</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="safeuexplorer.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="safeuexplorer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tvGlobal</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>CSafeUExplorer</receiver>
   <slot>sltItemClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twLocal</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CSafeUExplorer</receiver>
   <slot>sltLocalItemClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltItemClicked(QModelIndex)</slot>
  <slot>sltLocalItemClicked(QModelIndex)</slot>
 </slots>
</ui>
